src/Bicep.Core/TypeSystem/TypePropertyFlags.cs

```diff
@@ -33,8 +33,8 @@ public enum TypePropertyFlags
         WriteOnly = 1 << 3,
 
         /// <summary>
-        /// The property does not require inlining.
+        /// The property's value is resolved during deployment-time and does not require inlining.
```

Comment:
What about "This property's value known at the start of the deployment, and does not require inlining."